front-end/src/app/shared/models/transaction.model.ts

```diff
@@ -73,6 +73,7 @@ export abstract class Transaction extends BaseModel {
   memo_text_id: string | undefined;
 
   children: Transaction[] | undefined;
+  loan_agreement?: string;
```

Comment:
Can we name this loan_agreement_id? At first glance, I thought it held the transaction object for the loan agreement.